docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -88,6 +88,17 @@ panic threshold is 50%. This is :ref:`configurable <config_cluster_manager_clust
 runtime. The panic threshold is used to avoid a situation in which host failures cascade throughout
 the cluster as load increases.
 
+.. _arch_overview_load_balancing_priority_levels:
+
+Priority levels
+------------------
+
+During load balancing, Envoy will generally only consider hosts configured at the highest priority
+level. For each EDS :ref:`LocalityEndpoints<envoy_api_msg_LocalityLbEndpoints>` an optional priority
```

Comment:
You are going to hate me for being so picky on this short review, but s/LocalityEndpoints/LocalityLbEndpoints/.

Reply:
Not a bit - that's what reviews are for.
Thanks, and fixed.